### DISCUSSION TITLE
Looking for a maintainer

### DISCUSSION THREAD

**Discussion by kolypto** [kolypto/j2cli#83] [General]
This repository is looking for a maintainer. 

If you're ready to keep the project going and to dedicate a bit of your time for supporting it, please respond :)

**Comment by kpfleming** [comment]
This isn't what you asked for, but it's related: I created a fork at https://github.com/kpfleming/jinjanator. The major changes are:

- Supports all non-EOL versions of Python (including 3.12).
- Supports plugins (for input formats, tests, and filters).
- Has plugins for TOML and XML input.
- Has a plugin to make all of Ansible's tests and filters available.
- Full CI workflow including formatting/linting (with Ruff) and extensive unit tests.

This repository was a huge inspiration and I'm quite thankful that it existed, but I was fairly sure that radically changing nearly every part of the codebase (and build system) wasn't really something that the maintainer would be ready to accept.

**Comment by kolypto** [comment]
Hm, there are some more actively maintained forks :) 

* https://github.com/mattrobenolt/jinja2-cli
* https://github.com/m000/jj2cli

Perhaps I should just put some links on the README.md index page?
